### DISCUSSION TITLE
Use Wiki.js without defining page paths

### DISCUSSION THREAD

**Discussion by ohjohnsen** [requarks/wiki#4056] [Help / Questions]
I've used Wiki.js for a little while now.  I'm trying to find a good platform for documentation in my home, which I can use to collaborate with my wife.  But having to define paths for each page is a little cumbersome.  Is there a way to avoid having to define paths?  In short, just add pages with a title, and just use drag'n drop to move the page wherever I want it?

**Comment by Pingu-Pox** [comment]
I too am interested in this. It doesn't appear to be a possibility for as long as I've used wiki.js, but it'd be a feature that would attract more people for sure.

**Reply by ohjohnsen** [reply]
Yes, I think so as well.  Just look at Notion, and how you just add a page and drag it wherever when you feel like it.  At least for me, the structure of any documentation I write is subject to change many times.  So I think having to write out the path of a page when creating it is very impractical.

**Comment by ohjohnsen** [comment]
@NGPixel Would be very interested to hear what you have to say about this.  Is automatic path generation something what is planned for the future?  Or something that you think should be in the pipeline?

**Reply by NGPixel** [reply]
You won't have to define a path from the start in 3.0. However you will need to define it once you save it the first time. The ability to move and re-arrange entire "folders" is also planned later on.
